keras_hub/src/models/retinanet/retinanet_label_encoder.py

```diff
@@ -149,23 +146,25 @@ def _encode_sample(self, gt_boxes, gt_classes, anchor_boxes, image_shape):
             anchor_boxes: A Tensor with the shape `[total_anchors, 4]`
                 representing all the anchor boxes for a given input image shape,
                 where each anchor box is of the format `[x, y, width, height]`.
-            image_shape: Tuple indicating the image shape `[H, W, C]`.
+            height: int.
+            width: int.
 
         Returns:
             Encoded bounding boxes in the format of `center_yxwh` and
             corresponding labels for each encoded bounding box.
         """
-        anchor_boxes = convert_format(
+        anchor_boxes = keras.utils.bounding_boxes.convert_format(
             anchor_boxes,
             source=self.anchor_generator.bounding_box_format,
             target=self.bounding_box_format,
-            image_shape=image_shape,
+            height=height,
+            width=width,
         )
-        iou_matrix = compute_iou(
+        iou_matrix = keras.utils.bounding_boxes.compute_iou(
             anchor_boxes,
             gt_boxes,
             bounding_box_format=self.bounding_box_format,
-            image_shape=image_shape,
+            image_shape=(height, width, 3),
```

Comment:
Looks like the RetinaNetBackbone allows passing the number of channels, but here we are hardcoding it. Isn't that a bug?